backend/src/test/java/com/example/backend/service/ChatServiceTest.java

```diff
@@ -0,0 +1,275 @@
+package com.example.backend.service;
+
+import com.example.backend.models.ChatMessage;
+import com.example.backend.repo.ChatRepo;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.web.socket.CloseStatus;
+import org.springframework.web.socket.TextMessage;
+import org.springframework.web.socket.WebSocketSession;
+
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class ChatServiceTest {
+    @Mock
+    ChatRepo chatRepo;
+    @Mock
+    ObjectMapper objectMapper;
+    @Mock
+    WebSocketSession session;
+    @InjectMocks
+    ChatService chatService;
+
+    @Captor
+    private ArgumentCaptor<TextMessage> textMessageCaptor;
+
+    @BeforeEach
+    public void setUp() {
+        chatService = new ChatService(chatRepo, objectMapper);
+    }
+
+    @Test
+    void getSessions() {
+
+        ChatService spyChatService = spy(new ChatService(chatRepo, objectMapper));
+        Set<WebSocketSession> expectedSessions = new HashSet<>();
+        doReturn(expectedSessions).when(spyChatService).getSession();
+
+
+        Set<WebSocketSession> actualSessions = spyChatService.getSession();
+
+
+        assertEquals(expectedSessions, actualSessions);
+    }
```

Comment:
What exactly do you test here? You mock the getSession Method... So you are testing you mock?